cloud-init/freeipa-vars.tpl.yml

```diff
@@ -0,0 +1,9 @@
+---
+write_files:
+  - path: /var/lib/cloud/instance/freeipa-vars.sh
+    permissions: '0600'
```

Comment:
Just for my understanding, we don't need execute permissions for it to be run correctly?

Reply:
This file is `source`d, not run.

Reply:
Ah, right. Ok that makes complete sense.